src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.ts

```diff
@@ -5,11 +5,11 @@ import { ExclusionNavigation } from '../../../models/exclusion-navigation.interf
 import * as fromFeature from '../../../store';
 
 @Component({
-  selector: 'exui-describe-exclusion',
+  selector: 'exui-describe-exclusion-cc',
```

Comment:
the cc looks like a strange naming

Reply:
changed it now to exui-describe-exclusion-container